scarlet/display.py

```diff
@@ -285,15 +308,24 @@ def show_scene(
     return fig
 
 
+def get_extent(bbox):
+    extent = np.array([bbox.start[-1], bbox.stop[-1], bbox.start[-2], bbox.stop[-2]], dtype=float) - 0.5
```

Comment:
I don't think the -0.5 is necessary or even plausible

Reply:
This is needed when we set extent because of the way that matplotlib sets the grid with extent. Without the `0.5` the x's that mark the centers are located in the corner of a pixel instead of in the center. This is why I was against this change in matplotlib a few years ago.

Reply:
OK, I see. But the need for `origin` remains: https://matplotlib.org/3.1.1/tutorials/intermediate/imshow_extent.html